lib/cuckoo/common/demux.py

```diff
@@ -236,7 +244,9 @@ def demux_sample(filename: bytes, package: str, options: str, use_sflock: bool =
                 log.error("Detected password protected office file, but no sflock is installed: pip3 install -U sflock2")
 
     # don't try to extract from Java archives or executables
-    if "Java Jar" in magic or "PE32" in magic or "MS-DOS executable" in magic or any(x in magic for x in VALID_LINUX_TYPES):
+    if "Java Jar" in magic or "Java archive data" in magic or \
```

Comment:
Since we are converting OFFICE_TYPES to a constant, can we create JAVA_ARCHIVE_TYPES and EXECUTABLE_TYPES to representing these strings to?

Reply:
im gonna merge this, and those to types can be done later